en/development/testing.rst

```diff
@@ -1174,6 +1174,8 @@ token mismatches::
         $this->post('/posts/add', ['title' => 'Exciting news!']);
     }
 
+To make your FormProtector work properly in your integration tests, be aware that ``session_id('cli');`` needs to called in your /test/bootstrap.php.
```

Comment:
```suggestion
To make your FormProtector work properly in your integration tests, be aware that ``session_id('cli');`` needs to called in your ``/test/bootstrap.php``.
```

Reply:
shouldn't this be bootstrap_cli.php?

Reply:
@othercorey

> shouldn't` this be bootstrap_cli.php?
As the issue only affects my tests I would leave it in ``test/bootstrap.php``